airbyte-integrations/connectors/source-linkedin-pages/source_linkedin_pages/manifest.yaml

```diff
@@ -0,0 +1,841 @@
+version: 0.73.0
+type: DeclarativeSource
+
+definitions:
+  bearer_authenticator:
+    type: BearerAuthenticator
+    api_token: '{{ config[''credentials''][''access_token''] }}'
+
+  oauth_authenticator:
+    type: OAuthAuthenticator
+    client_id: '{{ config[''credentials''][''client_id''] }}'
+    client_secret: '{{ config[''credentials''][''client_secret''] }}'
+    refresh_token: '{{ config[''credentials''][''refresh_token''] }}'
+    refresh_request_body: { }
+    token_refresh_endpoint: 'https://www.linkedin.com/oauth/v2/accessToken'
+
+  selective_authenticator:
+    type: SelectiveAuthenticator
+    authenticator_selection_path: [ "credentials", "auth_method" ]
+    authenticators:
+      oauth2.0: "#/definitions/oauth_authenticator"
```

Comment:
this should be oAuth2.0 to be backwards compatible https://github.com/airbytehq/airbyte/pull/36744/files#diff-5d90e5e66a0037f6a4486c3e3ac857fbb3d29368a6b336f7faa06aed05e05121L116

Reply:
Fixed this